realm/realm-library/src/androidTest/java/io/realm/internal/OsResultsTests.java

```diff
@@ -0,0 +1,534 @@
+/*
```

Comment:
It seems that renaming this killed the git history so the old file is deleted and this is created. That doesn't seem ideal?